easybuild/easyconfigs/s/scikit-build-core/scikit-build-core-0.5.0-GCCcore-12.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-build-core'
+version = '0.5.0'
+
+homepage = 'https://scikit-build.readthedocs.io/en/latest'
+description = """Scikit-Build, or skbuild, is an improved build system generator
+for CPython C/C++/Fortran/Cython extensions."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+]
```

I think `CMake` is needed also at run-time:

```suggestion
builddependencies = [
    ('binutils', '2.40'),
    ('hatchling', '1.18.0'),
]

dependencies = [
    ('Python', '3.11.3'),
    ('Python-bundle-PyPI', '2023.06'),
    ('CMake', '3.26.3'),
]
```